static/admin/config.yml

```diff
@@ -1,6 +1,7 @@
 backend:
   name: git-gateway
-  branch: master
+  # branch: master
+  branch: studio-page-update
```

Comment:
I'm assuming this would be updated after review right?

Reply:
Yes, without it will take the `master` branch as back-end. Leading to incorrect data and build. Should be changed back to `master` AFTER *UAT* but we do need to let the others know

Reply:
Will keep it as is for QA/UAT - added a comment and mentioned the involved parties